src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -220,8 +224,10 @@ protected function describeContainerServices(ContainerBuilder $builder, array $o
 
         $tableHeaders = array_merge(array('Service ID'), $tagsNames, array('Class name'));
         $tableRows = array();
+        $rawOutput = isset($options['raw_output']) && $options['raw_output'];
```

Comment:
I think you're right - i've updated it. I used `raw_output` because that's what was used in `AbstractDescriptorText`. I've changed it to `raw_output` and updated that test to also request `raw_output`. I don't know a ton about this area of the code.